etc/osg/image-config.d/20-osg-ce-setup.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/bash
```

Comment:
One last thing -

I am not sure what invokes /etc/osg/image-config.d/20-osg-ce-setup.sh, but is it intended to be called directly?

In particular, I see that 50-nonroot-gratia-setup.sh has the execute bit set, but 20-osg-ce-setup.sh does not.

Reply:
Nope, `.sh` files are sourced by https://github.com/opensciencegrid/docker-software-base/blob/master/supervisord_startup.sh#L4 so we don't need the executable bit